.env-sample

```diff
@@ -0,0 +1,14 @@
+# Set to 'production' when deployed
+RACK_ENV=development
```

Comment:
*Shouldn’t* be necessary to set this at all in dev, I think. And Heroku will automatically set it.